src/cmd/cmd.zig

```diff
@@ -739,8 +744,16 @@ fn shredCollector() !void {
 
     // leader schedule
     // NOTE: leader schedule is needed for the shred collector because we skip accounts-db setup
-    var leader_schedule = try getLeaderScheduleFromCli(allocator) orelse @panic("No leader schedule found");
-    const leader_provider = leader_schedule.provider();
+    var leader_schedule_cache = LeaderScheduleCache.init(allocator, try EpochSchedule.default());
+
+    // This is a sort of hack to get the epoch of the leader schedule and then insert into the cache
```

Comment:
do we want to implement this before merging to main?

Reply:
I think the comment was outdated, the 'hack' is referring to how we get the epoch to insert the leader schedule and I think its fine for now